uiowa/uiowa
# Feature: Empty hamburger bug fix

Resolves https://github.com/uiowa/uiowa/issues/3828. 

# How to test

- Test all menu options, toggle, horizontal, mega with and without enabled menu items.    If there are no menu items, then the hamburger menu should not show for mobile or for the desktop toggle version.
